### ISSUE TITLE
Track example for students

### ISSUE THREAD

**Issue by ebbertd** [opencast/annotation-tool#374]
Sometimes instructors create tracks with example annotations to show students when to use which annotations. Instructors should be able to specify tracks which are always visible upon loading the annotation tool.

**Comment by ebbertd** [comment]
In a second step it would be desirable to add an export / import function for tracks.

**Comment by dagraf** [comment]
Our users did ask for the same feature. Here some specification we had in mind:

- Option «Show track all users» available under "Update/Edit track" > "Access" AND as own icon under "Change visibility
- Sub-options of "Show all users" (only appears when "Show all users" is selected):
   a) Writable: Users can make additions to the track, but not changes.
   b) Non-writable: Users cannot make any changes to the track.
- showing a track to all users can be undone.
- When the option «Show track all users» is checked, a copy of the selected track is created for all users.
- Changes made by Admins to the track displayed for all users are automatically synchronized to the users. This may require a reload of the browser.
- Offer Admin the option: how should the editable track be set in terms of public/privat/shared with Admin

**Comment by JulianKniephoff** [comment]
@dagraf 

First mockup of the UI parts:

![image](https://user-images.githubusercontent.com/123272/115189854-75d95900-a0e7-11eb-9286-55cc54d8ab3c.png)

![image](https://user-images.githubusercontent.com/123272/115189974-a28d7080-a0e7-11eb-9861-082d5eda4a06.png)

![image](https://user-images.githubusercontent.com/123272/115190156-e5e7df00-a0e7-11eb-81cf-61c865d8aac2.png)

![image](https://user-images.githubusercontent.com/123272/115190031-b5a04080-a0e7-11eb-8e92-89a3f0f9bb20.png)

In German it says "Allen Benutzern anzeigen."

By the way, shouldn't the English text be "Show **to** all users?"

**Comment by JulianKniephoff** [comment]
@dagraf do you have any preference about the order the tracks are shown in by default? The "example tracks" should probably go on top, right?

**Comment by dagraf** [comment]
> @dagraf do you have any preference about the order the tracks are shown in by default? The "example tracks" should probably go on top, right?
You're right: "Show to all users" is what it does. Please change english wording.

**Comment by dagraf** [comment]
Some specs/questions:

- Please change order: Shown to all users / Public / Shared with administrator _(please rename)_ / Private
- Would it be possible to add some explanation below the new option? For example: "The track and all containing annotations made by administrators will automatically be displayed to all users."

**Comment by dagraf** [comment]
Bug report: As a student, when I open a video containing an 'example track' in the annotation tool, this example track is not shown. I have to reload the browser, then it gets shown.
